backend/account_v2/authentication_controller.py

```
@@ -186,32 +187,49 @@ def set_user_organization(self, request: Request, organization_id: str) -> Respo
+                            organization_data.display_name,
+                            organization_data.id,
+                        )
+                        new_organization = True
+                        organization_member = self.create_tenant_user(
+                            organization=organization, user=user
+                        )
                 except IntegrityError:
                     raise DuplicateData(
                         f"{ErrorMessage.ORGANIZATION_EXIST}, \
                             {ErrorMessage.DUPLICATE_API}"
                     )
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)